main.go

```diff
@@ -1,19 +1,22 @@
 package main
 
 import (
-	"gorm.io/gorm"
+	//"errors"
+	"log"
+	//"net/http"
 	"github.com/gin-gonic/gin"
-	"net/http"
-
-
+	"github.com/gokhankocer/TODO-API/database"
+	"github.com/gokhankocer/TODO-API/handlers"
+	//"gorm.io/gorm"
 )
 
-type Todo struct{
-	ID string `json:"id"`
-	Status string `json:"status"`
-	Description string `json:"description"`
-}
-
 func main() {
-
-}
\ No newline at end of file
+	router := gin.Default()
+	database.Connect()
+	router.GET("/todos", handlers.GetTodos)
+	router.POST("todos", handlers.AddTodo)
+	router.DELETE("todos/:id", handlers.DeleteTodo)
+	router.GET("todos/:id", handlers.GetTodo)
+	router.PUT("todos/:id", handlers.UpdateTodo)
+	log.Fatal(router.Run("localhost:8080"))
```

Comment:
genelde port .env variabledan alınır başka bir port kullanmak için kod üzerinde değişiklik yapmadan kolayca değiştirebilmem lazım